### PR TITLE
PT-1199_remove_highwater

### PR DESCRIPTION
**Pull request by BigFab**
Everything about highwater and metric has been removed

### CODE REVIEW

**Review by ydrisR — DISMISSED** [review]
LGTM

File: api/hydrophoneApi.go [hunk 1]
```diff
@@ -77,20 +82,20 @@ func InitApi(
 	ntf clients.Notifier,
 	sl shoreline.Client,
 	gatekeeper commonClients.Gatekeeper,
-	metrics highwater.Client,
 	seagull commonClients.Seagull,
 	templates models.Templates,
 ) *Api {
+	logger := log.New(os.Stdout, CONFIRM_API_PREFIX, log.LstdFlags|log.Lshortfile)
```
Comment (ydrisR) [inline api/hydrophoneApi.go]: Why creating the logger with log.LstdFlags|log.Lshortfile flags?
It will always show the same line number (332)

**Review by ydrisR — COMMENTED** [review]
Looks good overall, just a question about the format of the log: why adding the line number information? It will always be the same...
And is it a good idea to prefix the whole line with "confirm/api"? Could it be a problem for log parsing later?

**Comment by BigFab** [comment]
@ydrisR concerning your comment to add the prefix "api/confirm" to all audit lines. I think that this will change anyway when we decide to not log audit on the stdout anymore. We can keep it like that or remove it, I am fine with both

**Review by ydrisR — DISMISSED** [review]
LGTM

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ This API sends notifications to users for things like forgotten passwords, initi
 - PT-995: Document the api using openApi and swaggo
 ### Changed
 - PT-414 Add switch to enable/disable sending of emails
+- PT-1199 Remove Highwater (replace by log audits)
```
Comment (ydrisR) [inline CHANGELOG.md]: This should be under 0.8.1 section :(

**Review by ydrisR — APPROVED** [review]
OK

**Pull request merged**